### PR TITLE
PR to re-enable eslint-diff

### PR DESCRIPTION
**Pull request by burtonator**
## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/7391

## Description of Changes
- changed the eslint 'epoch' to today
- add instructions on how to disable.

## "How We Fixed It"
- implemented merge queues.

## Test Plan
- there is a lint canary that tests things now.
- 
## Deployment Plan
- it's just in CI/CI

## Other Considerations

-

### CODE REVIEW

**Review by Rotorsoft — APPROVED** [review]

**Review by ForestMars — APPROVED** [review]
I'm good with this, however with @kurtisassad OOO, would like to get @timolegros approval here before we merge.

**Review by timolegros — APPROVED** [review]

**Pull request merged**
